### ISSUE TITLE
Add reserves info to fee method

### ISSUE THREAD

**Issue by mDuo13** [XRPLF/rippled#4477]
The [`fee` method](https://xrpl.org/fee.html) provides info about transaction costs/fees but doesn't tell you about the cost of reserves, which is similarly useful information often used together. In particular, the transaction cost for an AccountDelete (and, in XLS-30d, AMMCreate) is one owner reserve increment, so it would be useful to know this info for setting the `Fee` of those transactions.

One can get this from server_info/server_state now, but those methods are more work for the server than `fee`.

**Comment by riken127** [comment]
Hi! I'd like to take a stab at this. Is it still open for contributions?

**Comment by mvadari** [comment]
> Hi! I'd like to take a stab at this. Is it still open for contributions?

Yes, this is an open source repo that accepts contributions from all.
